bootstrap/src/lib/cache/index.ts

```diff
@@ -28,6 +34,12 @@ export const defaultOptions = () => ({
       ...(env.CACHE_KEY_IGNORED_PROPS || '').split(',').filter((k) => k), // no empty keys
     ],
   },
+  rateLimit: {
+    groupMaxAge: parseInt(env.GROUP_MAX_AGE || '') || DEFAULT_GROUP_MAX_AGE,
+    groupId: (env.API_KEY && hash(env.API_KEY)) || '',
+    id: DEFAULT_CACHE_KEY_RATE_LIMIT_PARTICIPANT,
```

Comment:
We should not need this anymore, right? There are no RL groups or participants anymore. We have one RL instance per adapter that keeps track of multiple request types.

Reply:
Yep, removed already